### DISCUSSION TITLE
Sonoma 14.1 on ssd

### DISCUSSION THREAD

**Discussion by sprocj** [community/community#73141] [Programming Help]
I have installed Sonoma 14.1 on my imac 2017 and it works fine one query I have when I turn on computer the apple logo shows then the ssd shows and stalls for a few seconds before moving to progress bar is there anyway of using terminal to not show ssd at startup

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @sprocj ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by davea42** [comment]
If this is still bothering you maybe you could take a photo of the screen (with phone) while
the ssd is showing and then post the photo here.   
Make sure  the image is sensibly sized, not  Megabytes in size.
That way we might have an idea as to what is going on.
